hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/audit/OMAction.java

```diff
@@ -60,6 +60,7 @@ public enum OMAction implements AuditAction {
   RENEW_DELEGATION_TOKEN,
   CANCEL_DELEGATION_TOKEN,
   GET_SERVICE_LIST,
+  TRANSFER_LEADERSHIP,
```

Comment:
I would suggest making transfer leadership action consistent in OMAction and SCMAction. The current naming is:

- OMAction.TRANSFER_LEADERSIP
- SCMAction.TRANSFER_SCM_LEADER

Reply:
Got it, I will unify them!